sp/src/game/shared/hl2/hl2_gamerules.cpp

```diff
@@ -2118,6 +2151,45 @@ ConVar  alyx_darkness_force( "alyx_darkness_force", "0", FCVAR_CHEAT | FCVAR_REP
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_BULLSQUID,		D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_CITIZEN_PASSIVE,	D_HT, 0 );	
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_CITIZEN_REBEL,	D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_COMBINE,			D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_COMBINE_GUNSHIP,	D_FR, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_COMBINE_HUNTER,	D_FR, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_CONSCRIPT,		D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_FLARE,			D_NU, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_HEADCRAB,			D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_HOUNDEYE,			D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_MANHACK,			D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_METROPOLICE,		D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_MILITARY,			D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_MISSILE,			D_NU, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_SCANNER,			D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_STALKER,			D_NU, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_VORTIGAUNT,		D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_ZOMBIE,			D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_PROTOSNIPER,		D_FR, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_EARTH_FAUNA,		D_NU, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_PLAYER_ALLY,		D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_PLAYER_ALLY_VITAL,D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_HACKED_ROLLERMINE,D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_ALIEN_PREDATOR,	D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_ALIEN_FAUNA,		D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_ALIEN_FLORA,		D_NU, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_RACE_X,			D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_COMBINE_NEMESIS,	D_HT, 0 );
+		CBaseCombatCharacter::SetDefaultRelationship( CLASS_COMBINE_HUSK,			CLASS_ARBEIT_TECH,		D_NU, 0 );
```

Comment:
Do husks need a relationship defined with themselves?

Reply:
I actually forgot about this 😬

I think they might already be set to `D_NU` by default. They shouldn't like each other because I think that would enable friendly fire prevention on default soldiers, so I'll add an explicit definition for `D_NU`.